benchmarks/helics/helics_benchmark_main.h

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright (c) 2017-2019,
+Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance for Sustainable Energy, LLC.  See
+the top-level NOTICE for additional details. All rights reserved.
+SPDX-License-Identifier: BSD-3-Clause
+*/
+#pragma once
+#include <benchmark/benchmark.h>
+#include "helics/helics-config.h"
+#include <iostream>
+#ifdef ENABLE_ZMQ_CORE
+#include "helics/core/zmq/ZmqCommsCommon.h" 
+#endif
+
+// Helper macro to create a main routine in a test that runs the benchmarks
+#define HELICS_BENCHMARK_MAIN()                                                                                   \
+    int main (int argc, char **argv)                                                                              \
+    {                                                                                                             \
+        ::benchmark::Initialize (&argc, argv);                                                                    \
+        if (::benchmark::ReportUnrecognizedArguments (argc, argv))                                                \
+            return 1;                                                                                             \
+        printHELICSsystemInfo ();                                                                                 \
+        ::benchmark::RunSpecifiedBenchmarks ();                                                                   \
+    }                                                                                                             \
+    int main (int, char **)
```

Comment:
Is this extra `int main (int, char **)` needed for something?

Reply:
I copied it from the google benchmark BENCHMARK_MAIN macro.  I think they put that there so you can put a ';' after the macro